### PR TITLE
SM Logic A108-110

### PR DESCRIPTION
**Pull request by Logarhythmic**
https://docs.google.com/spreadsheets/d/1s04k5ZuvCV28-ZJtZbVghlfMCg0Szr6hh69tFAjXv3E/edit#gid=444856701

### CODE REVIEW

File: randovania/games/super_metroid/logic_database/Maridia.txt [hunk 1]
```diff
@@ -112,7 +112,40 @@ Extra - asset_id: 126
...
+                  # IBJ
+                  Vertical Infinite Bomb Jump (Intermediate) and Can Lay Bombs
+              Any of the following:
+                  # Through the Morphball Tunnel
+                  All of the following:
+                      # With Morph
+                      Morph Ball
+                      Any of the following:
+                          Mid-Air Morph (Intermediate)
+                          Ice Beam and Ice Beam Abuse (Advanced) and Can Use Spring Ball
```
Comment (Miepee) [inline randovania/games/super_metroid/logic_database/Maridia.txt:133]: This should get a comment

File: randovania/games/super_metroid/logic_database/Maridia.txt [hunk 2]
```diff
@@ -112,7 +112,40 @@ Extra - asset_id: 126
...
+                  # Without Morph: Clip with Ice Beam Abuse
```
Comment (Miepee) [inline randovania/games/super_metroid/logic_database/Maridia.txt:134]: some video or further description would be helpful

File: randovania/games/super_metroid/logic_database/Maridia.txt [hunk 3]
```diff
@@ -112,7 +112,40 @@ Extra - asset_id: 126
...
+                  Ice Beam and Ice Beam Abuse (Expert) and Knowledge (Intermediate)
+                  # Springball Jump
+                  Hi-Jump Boots and Spring Ball Jump (Intermediate) and Can Use Spring Ball
+          All of the following:
+              # Without Graivty: https://youtu.be/WNYN6kMkAkI
```
Comment (Miepee) [inline randovania/games/super_metroid/logic_database/Maridia.txt:139]: typo in graivty

File: randovania/games/super_metroid/logic_database/Maridia.txt [hunk 4]
```diff
@@ -112,7 +112,40 @@ Extra - asset_id: 126
...
+              Ice Beam and Continuous Wall Jump (Advanced) and Ice Beam Abuse (Expert)
```
Comment (Miepee) [inline randovania/games/super_metroid/logic_database/Maridia.txt:140]: this needs a bit better of a description

File: randovania/games/super_metroid/logic_database/Maridia.txt [hunk 5]
```diff
@@ -516,17 +556,39 @@ Extra - asset_id: 141
...
   * Normal Door to Mt. Everest/Door to Crab Shaft
   > Door to Pseudo Plasma Spark Room
-      All of the following:
-          Gravity Suit
-          Hi-Jump Boots or Space Jump
+      Any of the following:
+          All of the following:
+              # With Gravity
+              Gravity Suit
+              Any of the following:
```
Comment (Miepee) [inline randovania/games/super_metroid/logic_database/Maridia.txt:563]: what about space jump and HJ?

File: randovania/games/super_metroid/logic_database/Maridia.txt [hunk 6]
```diff
@@ -209,6 +242,13 @@ Extra - asset_id: 129
 > Door to Crab Hole; Heals? False
   * Layers: default
   * Normal Door to Crab Hole/Door to Crab Tunnel
+  > Door to Main Street
+      All of the following:
+          # GGG
```
Comment (Miepee) [inline randovania/games/super_metroid/logic_database/Maridia.txt:247]: please use full name

File: randovania/games/super_metroid/logic_database/Maridia.txt (hunk 2 above)
Comment (Logarhythmic) [reply randovania/games/super_metroid/logic_database/Maridia.txt:134]: The youtube video in the comment at the top of this AND tree shows the trick in full. Each of these additional requirements make it slightly different, but they're all different versions of that same tech.

Comment (Miepee) [reply randovania/games/super_metroid/logic_database/Maridia.txt:135]: i dont see a yt vide at the top of this AND.

Comment (Logarhythmic) [reply randovania/games/super_metroid/logic_database/Maridia.txt:136]: Added the video and timestamped the ice clip

**Review by Miepee — APPROVED** [review]

**Pull request merged**